README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
```

Comment:
Referring to the TSC as a "home" for "projects and assets" sounds slightly confusing to me. Did you mean the Foundation rather than the TSC? Since most of the language is repeated later in this paragraph, I would skip this sentence here and begin the paragraph with "The mission of the Node.js Foundation..."